src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/note/NoteMapper.kt

```diff
@@ -55,13 +54,6 @@ object NoteMapper {
       EntityNoteType.GOAL_COMPLETION -> DomainNoteType.GOAL_COMPLETION
     }
 
-  fun toResourceModel(noteType: DomainNoteType) =
```

Comment:
This method is now in my new mapper
To be fair this method probably shouldn't have been here (my bad, I should have seen it at PR time) because it is mapping a domain type into a REST resource type; whereas this mapper class is in the jpa package so should not be concerned with REST API stuff! 😁